### PR TITLE
Switched to building frontend modules in serial rather than parallel

### PR DESCRIPTION
**Pull request by rohan-bes**
Building in parallel caused out of memory errors on feature instances

### CODE REVIEW

File: packages/web-frontend/package.json [hunk 1]
```diff
@@ -12,7 +12,7 @@
   "main": "dist/index.js",
   "scripts": {
     "analyze": "source-map-explorer build/static/js/main.*",
-    "build": "rm -rf dist && run-p \"build:* {@}\" --",
+    "build": "rm -rf dist && run-s \"build:* {@}\" --",
```
Comment (rohan-bes) [inline packages/web-frontend/package.json:15]: We are still using `run-p` for most other `build` commands, as they're just building types and source code. Figured it was safe to leave them, but an argument could be made to switch to `run-s` for all I suppose...

**Review by nick-beyondessential — APPROVED** [review]
LGTM

File: scripts/bash/getDeployablePackages.sh [hunk 2]
```diff
@@ -1,4 +1,4 @@
 #!/bin/bash -e
 
-echo "admin-panel" "lesmis" "psss" "web-frontend" "admin-panel-server" "entity-server" "lesmis-server" "meditrak-server" "psss-server" "report-server" "web-config-server"
+echo "admin-panel" "lesmis" "psss" "web-frontend" "report-server" "admin-panel-server" "entity-server" "lesmis-server" "meditrak-server" "psss-server" "web-config-server"
```
Comment (rohan-bes) [inline scripts/bash/getDeployablePackages.sh:3]: Turns out this is also an issue. Now `admin-panel-server` depends on `report-server` so they need to be built in the correct order. I think when we get `lerna` these kinds of issues will resolve themselves, right @IgorNadj ?

Comment (IgorNadj) [reply scripts/bash/getDeployablePackages.sh:3]: Yeah there's a few circular dependancies at the moment which are untangled in the followup PR.

**Pull request merged**